pkgs/tools/networking/litmus/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  version = "0.13";
+  name = "litmus-${version}";
```

Comment:
Please use ``pname``.

Reply:
Is there any documentation about name vs. pname? I cannot find anything in the documentation.

Reply:
https://github.com/NixOS/rfcs/pull/35, perhaps?

Reply:
Thanks. But when I understand RFC0035 correctly, it only says that ``name`` should be automatically set to ``${pname}-${version}``, if ``pname`` and ``version`` are set, to reduce redundancy. (But it seems that noone notices the downside of this -- it makes package-definitons less obvious, since then ``name`` is "magically" defined and comes out of nowhere. Explicit is better than implicit.)
But RFC0035 does neither require nor recommend to use ``pname``, so the above is perfectly fine.

If "you should use pname instead of name" is the official NixOS-optionion, it should be mentioned in the documentation.

Reply:
> If "you should use pname instead of name" is the official NixOS-optionion, it should be mentioned in the documentation.


You are right, feel free to open a pull request for the documentation to include it.

Reply:
No, I won't and I can't document it.
It's the duty of the ones who want to promote ``pname`` to document it, including why and how exactly to use it.